src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -22,12 +22,42 @@ public static long countWords() throws IOException {
         return count;
     }
 
-    public static void printLongestWordTop100() throws IOException {
+    public static long countWordsStream() throws IOException {
         String contents = new String(Files.readAllBytes(Paths
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        // TODO 이 부분에 구현한다.
+        long count = words.stream().filter(word -> word.length() > 12).count();
+
+        return count;
+    }
+
+    public static List<String> printLongestWordTop100() throws IOException {
+        String contents = new String(Files.readAllBytes(Paths
+                .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
+        List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
+
+        List<String> over12lengthList = words.stream()
+                .filter(word -> word.length() > 12)
+                .collect(Collectors.toList());
+
+        List<String> sortedReverseLengthList = over12lengthList
+                .stream()
+                .sorted((s1, s2) -> -1 * Integer.compare(s1.length(), s2.length()))
```

Comment:
```suggestion
             .sorted(Comparator.comparing(String::length).reversed())
```

String은 Comparable 을 구현하고 있어 비교가 가능합니다.